File: 0657-robot-return-to-origin/0657-robot-return-to-origin.go
func judgeCircle(moves string) bool {
    move_op := make(map[string]int)
    for i:=0; i<len(moves); i++{
        move_op[string(moves[i])] += 1
    }
    return (move_op["U"] == move_op["D"]) && (move_op["L"] == move_op["R"])
}

File: 0657-robot-return-to-origin/0657-robot-return-to-origin.py
import collections

class Solution:
    def judgeCircle(self, moves: str) -> bool:
        move_op = collections.defaultdict(lambda:0)
        for move in moves:
            move_op[move] += 1
        return (move_op["L"] == move_op["R"]) and (move_op["U"] == move_op["D"])
        